zer0-os/zOS
# feat: match conversation search inputs to Figma

### What does this do?

Uses `zUI` `0.24.0` `Input` to match both inputs to Figma.

### Why are we making this change?

Inputs didn't match Figma.

### How do I test this?

1. Open app
2. Open chat sidemenu
3. Check the search bar matches Figma
4. Hit "Create Zero Message"
5. Check the search bar matches Figma

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
